pkg/cloud/packet/client.go

```diff
@@ -138,25 +140,42 @@ func (p *Client) NewDevice(ctx context.Context, req CreateDeviceRequest) (*packn
 
 	userData = stringWriter.String()
 
-	// Allow to override the facility for each PacketMachineTemplate
-	facility := req.MachineScope.PacketCluster.Spec.Facility
-	if req.MachineScope.PacketMachine.Spec.Facility != "" {
-		facility = req.MachineScope.PacketMachine.Spec.Facility
+	// If Metro or Facility are specified at the Machine level, we ignore the
```

Comment:
This is how I'm thinking the logic would flow:
```go
// Allow the user to override the facility and metro for each PacketMachineTemplate, default to the PacketCluster values
facility := packetClusterSpec.Facility
metro := packetClusterSpec.Metro

// do we need the check below? didn't we do this already when the resource was updated?
if metro > "" && facility > "" {
   // return an error, you can't have both in a cluster definition
}

if packetMachineSpec.Facility > "" {
  // do we need the check below? didn't we do this already when the resource was updated?
  if packetMachineSpec.Metro > "" {
     // return error, you can't have both facility and metro in machine definition
  }
  
  facility = packetMachineSpec.Facility
  // machine setting trumps cluster setting
  metro = ""
} else if packetMachineSpec.Metro > "" {
  metro = packetMachineSpec.Metro
  // machine setting trumps cluster setting
  facility = ""
}
```

Reply:
So we did decide you can specify both but if you did metro would be used and facility ignored.